services/user.py

```diff
@@ -0,0 +1,44 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(username: str, password: str, email: str = None,
+                first_name: str = None, last_name: str = None) -> None:
```

Comment:
Place each argument on a new line, with proper indentation and formatting.

Reply:
agree, just improve reading 💅🏻